linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -407,8 +408,12 @@ fn direct_credit_message(recipient: ChainId, amount: Amount) -> OutgoingMessage
 fn generate_key_pairs(count: usize) -> Vec<AccountSecretKey> {
     let mut key_pairs = iter::repeat_with(Secp256k1SecretKey::generate)
         .map(AccountSecretKey::Secp256k1)
-        .take(count)
+        .take(count - 2)
         .collect::<Vec<_>>();
+    key_pairs.extend(vec![
+        AccountSecretKey::EvmSecp256k1(EvmSecretKey::generate()),
+        AccountSecretKey::EvmSecp256k1(EvmSecretKey::generate()),
+    ]);
```

Comment:
What if `count < 2`? Maybe just do a single iteration and alternate the variants instead?